src/main/webapp/app/pool/pool-booking/pool-booking.component.ts

```diff
@@ -10,16 +11,44 @@ import { PoolBookingService } from 'app/pool/pool-booking/pool-booking.service';
 export class PoolBookingComponent implements OnInit {
     public reservationPoolId: string;
     public count: number;
+    tools: Tool[];
+    public selectedTools: string[];
 
-    constructor(private route: ActivatedRoute, private poolBookingService: PoolBookingService) {}
+    constructor(private route: ActivatedRoute, private poolBookingService: PoolBookingService, private elementRef: ElementRef) {}
 
     ngOnInit(): void {
         this.route.paramMap.subscribe(e => (this.reservationPoolId = e.get('id')));
         this.count = 1;
+        this.poolBookingService.getTools(this.reservationPoolId).subscribe(tools => {
+            this.tools = tools.body;
+            this.createToolList();
+        });
+        this.selectedTools = [];
+    }
+
+    private createToolList(): void {
+        const select = this.elementRef.nativeElement.querySelector('#toolsSelected');
+        for (const tool of this.tools) {
+            const option = '<option>' + tool.name + '-' + tool.version + '</option>';
```

Comment:
It's angular. We have things like `*ngFor` and `ngModel`. Thats not the way it is done. 

it works like this: [ngFor](https://angular.io/guide/displaying-data)